src/assets/wise5/services/teacherProjectService.ts

```diff
@@ -2574,9 +2574,11 @@ export class TeacherProjectService extends ProjectService {
    */
   addGroupChildNodesToInactive(node) {
     for (const childId of node.ids) {
-      const childNode = this.getNodeById(childId);
-      this.project.inactiveNodes.push(childNode);
-      this.inactiveStepNodes.push(childNode);
+      if (!this.isInactive(childId)) {
```

Comment:
Can we use isActive() to avoid double negation?
```suggestion
      if (this.isActive(childId)) {
```

Reply:
Looks like using isActive() doesn't work because at that point in time the steps have been taken out of the active nodes array so isActive() would return false and then the steps would not get added to the inactive nodes array even when they should. This would specifically break the action of moving an active lesson to the unused lessons section.

Reply:
I see. So this is a state where the node is NOT active, and also is NOT inactive? Like in-between? Maybe we should have a function that checks for that instead? When I read that "X is not inactive", I take it to mean that "X is active".